up-l1/README.adoc

```diff
@@ -403,11 +478,12 @@ uProtocol defines bindings to the following communication protocols:
 * link:http.adoc[*HTTP*]
 * link:someip.adoc[*SOME/IP*]
 
-Each uProtocol Client *MUST* employ exactly one of these bindings for implementing the Transport Layer API.
-
-Additional bindings *MAY* be defined in future versions of uProtocol.
 
-A binding *MAY* employ link:https://cloudevents.io/[CloudEvents] as a means to map uProtocol messages to the communication protocol's PDU. In order to provide for consistency across implementations, such bindings *MUST* adhere to link:cloudevents.adoc[*UMessage mapping to CloudEvents*]
+If a communication protocol binding uses link:https://cloudevents.io/[CloudEvents]:
 
+[.specitem,oft-sid="dsn~utransport-cloudevents~1",oft-needs="impl,utest"]
```

Comment:
a transport that does not use CloudEvents will not be able to cover this spec item

Reply:
Yes and that is why there is a "If" at the beginning of line 482

Reply:
Given that we have moved away from CloudEvents, I would prefer removing the requirements entirely as non of the transports will use it in version >1.5.